modules/ROOT/partials/misc/purchase-premium-plugins.adoc

```diff
@@ -15,32 +29,47 @@ Try the *{pluginname} {extensionType}* and the {cloudname} with a free link:{acc
 
 === Get {pluginname}
 
-ifeval::[{pluginminimumplan} != 'enterprise']
+ifndef::isEnterprise[]
 The *{pluginname} {extensionType}* {isAre} included in the following {cloudname} plans:
 endif::[]
-ifeval::[{pluginminimumplan} == 'enterprise']
+ifdef::isEnterprise[]
 The *{pluginname} {extensionType}* {isAre} included in link:{pricingpage}[{enterpriseplan}].
 endif::[]
 
-{% case pluginminimumplan %}
-{% when 'tierone' %}- The link:{pricingpage}[{tieroneplan}].
-{% when 'tierone' or 'tiertwo' %}- The link:{pricingpage}[{tiertwoplan}].
-{% when 'tierone' or 'tiertwo' or 'tierthree' %}- The link:{pricingpage}[{tierthreeplan}].
-- link:{pricingpage}[{enterpriseplan}].
-{% endcase %}
+ifdef::isTierOne[]
+* The link:{pricingpage}[{tieroneplan}].
+endif::[]
+ifdef::isTierOne,isTierTwo[]
+* The link:{pricingpage}[{tiertwoplan}].
+endif::[]
+ifdef::isTierOne,isTierTwo,isTierThree[]
+* The link:{pricingpage}[{tierthreeplan}].
+endif::[]
+ifndef::isEnterprise[]
+* link:{pricingpage}[{enterpriseplan}].
+endif::[]
 
-{% if pluginminimumplan == 'tiertwo' or pluginminimumplan == 'tierthree' %}
+ifdef::isTierTwo[]
 A 14-day free trial is also available for the {tiertwoplan} and the {tierthreeplan}.
-{% elsif pluginminimumplan == 'tierthree' %}
+endif::[]
+ifdef::isTierThree[]
 A 14-day free trial is available for the {tierthreeplan}.
-{% endif %}
+endif::[]
 
 == Further information
 
 For information on:
 
-* {cloudname} plans, see: link:{pricingpage}[{companyname} Pricing Plans].{% if extensionType == 'Packs' %}
-* Enabling premium skins, see: link:{baseurl}/configure/editor-appearance/#skin[The `skin` setting].
-* Enabling premium icon packs, see: link:{baseurl}/configure/editor-appearance/#icons[The `icons` setting].{% else %}
-* Configuring the *{pluginname} {extensionType}*, see: link:{baseurl}/plugins/premium/{plugindocspage}/[The {pluginname} {extensionType} documentation].
-{% endif %}
+* {cloudname} plans, see: link:{pricingpage}[{companyname} Pricing Plans].
+ifeval::["{extensionType}" == "Packs"]
+* Enabling premium skins, see: xref:configure/editor-appearance.adoc#skin[The `skin` setting].
+* Enabling premium icon packs, see: xref:configure/editor-appearance.adoc#icons[The `icons` setting].
+endif::[]
+ifeval::["{extensionType}" != "Packs"]
+* Configuring the *{pluginname} {extensionType}*, see: xref:plugins/premium/{plugindocspage}[The {pluginname} {extensionType} documentation].
+endif::[]
+
+:!isTierOne:
```

Comment:
```suggestion
:!isAre:
:!isTierOne:
```
We probably should unset `isAre` here as well since it's defined within this file.

Reply:
done